homework1/task02/tests/test_fib.py

```diff
@@ -0,0 +1,21 @@
+from typing import List
+
+import pytest
+from fibonacci.fibonacci import check_fibonacci
+
+
+@pytest.mark.parametrize(
+    ("lst", "expected_result"),
+    [
```

Comment:
Please check that you have cases for a list of sizes: 1,2,3, 4  both fib and not fib.

I'd recommend having two tests: fib and not fib, in that case, you can remove expected_result from parameters.

Reply:
Yes, good idea. Done